### ISSUE TITLE
Doesn't work with the Journals Plugin

### ISSUE THREAD

**Issue by polyinnovator** [mirnovov/obsidian-homepage#92]
### Summary

The journals plugin has replaced both the calendar by cain and the periodic notes plugins for me (and a lot others actually). As it is a superior experience.

However I wasn't able to tell this plugin to choose daily or weekly notes, as it doesn't sense that it has them. Even though it does.

**Comment by mirnovov** [comment]
It sounds like the Journals plugin is going to have a major revamp, cf. srg-kostyrko/obsidian-journal#54. So I'm happy to add this functionality, but I'd rather wait until srg-kostyrko has revamped his plugin, so I'm redoing a whole bunch of stuff just after implementing it.

**Comment by polyinnovator** [comment]
Yeah totally! I didn't know he was doing major work to it tbh, I just have used all of the calendar plugins, and found this one to be the best. (Even out of the big ones haha)

You're right to wait until after!
    On Saturday, July 13, 2024 at 10:40:59 PM CDT, mirnovov ***@***.***> wrote:  
 
 


It sounds like the Journals plugin is going to have a major revamp, cf. srg-kostyrko/obsidian-journal#54. So I'm happy to add this functionality, but I'd rather wait until srg-kostyrko has revamped his plugin, so I'm redoing a whole bunch of stuff just after implementing it.

**Comment by mirnovov** [comment]
It seems v2 has been released now, so putting this on my to do list.
